### DISCUSSION TITLE
A few complaints about the Pupil Player interface

### DISCUSSION THREAD

**Discussion by jonmatthis** [pupil-labs/pupil#2291] [Ideas]
1) The basic design where everything is overlaid on top of the world video image is problematic. It has always been a pain (leading to a cluttered work space), but now that Offline Calibration is implemented it has become more of a problem. In particular, it is impossible to click on the lower edge of the screen during the Natural Features calibration, as that portion is blocked by the playhead and other graphs. Even when that section is minimized as much as possible, it still blocks a good portion of the screen. 

I imagine that reworking the entire system to use dockable windows (similar to Adobe products, like Illustrator), but please at least make it *possible* to interact with all of the world video screen during Natural Features calibration. 

2) On a similar note, please make it possible to turn off some of the new timeline data streams (i.e. Pupil Diameter, recorded FPS, etc). They are not useful to everyone, and add to the clutter on the screen, which is distracting. I would love to be able to toggle these graphs on or off in the System Graphs window. 

3) Related, while you're at it, you should also add an option to show a trace norm_pos_x and norm_pos_y to the timeline. 

4) In fact, while we're at THAT, I'd love an option to overlay the norm_pos_x and norm_pos_y trace on the output world_viz video (similar to the Eye Video Overlay option), but maybe that's just me. 

That's all for now, thanks for letting me vent!

**Comment by jonmatthis** [comment]
Oh! And it would also be very helpful to have an option to overlay the frame number onto the world_viz output video.

**Comment by mkassner** [comment]
@papr probably loves to hear this :-) (He has been advocating for these things for a while.)

Thanks you for the feedback. We certainly plan to implement it!

**Comment by willpatera** [comment]
@jonmatthis thanks for the feedback :smile: We are working on improving the interface and will look to address your feedback with forthcoming releases.
